ersilia/core/tracking.py

```diff
@@ -3,26 +3,85 @@
 import sys
 import csv
 import json
+import time
 import boto3
+import docker
 import shutil
+import psutil
 import logging
 import requests
 import tempfile
+import resource
 import statistics
 import tracemalloc
+from .session import Session
 from datetime import datetime
 from .base import ErsiliaBase
 from collections import defaultdict
 from ..default import EOS, ERSILIA_RUNS_FOLDER
-from ..io.output_logger import TabularResultLogger 
+from ..io.output_logger import TabularResultLogger
 from botocore.exceptions import ClientError, NoCredentialsError
 
 
-# Temporary path to log files until log files are fixed
-TEMP_FILE_LOGS = os.path.abspath("")
+def docker_stats(container_name=None):
+    """
+    This function will calculate the memory usage of the Docker container running Ersilia Models.
```

Comment:
This docstring is misleading. You are calculating both memory and cpu consumption which is not reflected here. :)